### PR TITLE
OPNET-219: Add dev-scripts ci job for v6-primary dual stack

### PR DESCRIPTION
**Pull request by cybertron**

### CODE REVIEW

**Comment by cybertron** [comment]
Hmm, we seem to be default to 4.12 still for some reason. I thought we had bumped that. This won't work until 4.13 so that will have to resolved before this can go in.

/hold

**Comment by cybertron** [comment]
/pj-rehearse

Let's see if these jobs have moved to 4.13 yet.

**Comment by cybertron** [comment]
/pj-rehearse

I want to see if we get the same set of failures again. It's not clear to me that all of them are related to this new type of env.

**Comment by cybertron** [comment]
/pj-rehearse

Timed out, which is a new failure mode. Since the only change I made was to skip some tests it seems unlikely this is a problem with the PR.

**Comment by cybertron** [comment]
Argh, two of the test skips got combined, so both tests got run. A little surprised by the DNS failures, but they seem to have passed before so I'm going to see if that's a temporary thing.

**Comment by cybertron** [comment]
/pj-rehearse

image registry problem should be fixed

**Comment by cybertron** [comment]
/hold cancel

Passing, modulo the skips. Would like to get this in so we can catch regressions in this type of environment.

File: ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml [hunk 1]
```diff
@@ -73,6 +73,29 @@ tests:
...
     workflow: baremetalds-e2e-ovn-dualstack
+- as: e2e-metal-ipi-ovn-dualstack-v6
+  cluster: build05
+  optional: true
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6v4
+      TEST_SKIPS: should run iperf2 \| CVE-2021-29923 \| pod containing more than
```
Comment (andfasano) [inline ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml:84]: @cybertron are these skips meant to be temporary? If not I'd expected them to be filtered in `openshift/origin` (ie https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go)

Comment (cybertron) [reply ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml:84]: Yes, they're temporary. I have a story on our backlog to look into why they're failing, but I expect we should fix all of them eventually.

https://issues.redhat.com/browse/OPNET-337

**Comment by cybertron** [comment]
/pj-rehearse

It's been a while so I'd like to see if this still passes.

**Comment by cybertron** [comment]
/hold

Failure looks possibly legit. Will try this locally to see if it reproduces.

**Comment by cybertron** [comment]
/pj-rehearse

v6-primary is working for me locally again. Let's see if it does in ci too.

**Comment by cybertron** [comment]
/hold cancel
/pj-rehearse

For some reason it didn't retrigger. Removing the hold since the known blocker has been resolved.

**Comment by cybertron** [comment]
Failed on an infra issue. I think there were some changes in how metal ci envs are provisioned recently, so I'm going to rebase this and see if that gets it working.

**Comment by cybertron** [comment]
/pj-rehearse

One more time before I go ask for help.

**Comment by cybertron** [comment]
/pj-rehearse

Trying again so I can get on the env and poke around while it's running.

**Comment by cybertron** [comment]
/retest-required

Test pod seems to have timed out for some reason.

**Comment by cybertron** [comment]
/pj-rehearse

A possible fix for the failure I'm seeing merged this morning.

**Comment by cybertron** [comment]
/pj-rehearse

Let's see if this passes now that 4.14 is out.

**Comment by cybertron** [comment]
/pj-rehearse

dev-scripts is now defaulting to 4.15. Let's see if that gets this working.

**Comment by cybertron** [comment]
/pj-rehearse

I don't think the last run even got to the point of trying the deployment.

**Comment by cybertron** [comment]
/pj-rehearse ack

We know the job is accurately representing the state of the feature in the repo, so let's get this job in so we can iterate on it without a round trip to the release repo.

**Pull request merged**